### PR TITLE
Ensure cypress tests are passing after accounting to changes in dom e…

### LINKED ISSUE #35: Refactor - Cypress Testing
Cypress tests need to be updated to account for the new pet images rendered to the DOM.

### PR DESCRIPTION
**Pull request by paulina-isabel**
Closes #35 

What I did:
Refactor Cypress tests to account for changed made in pet details page

Testing Notes/Extras: 

Did this break anything?
- [x]  No
- [ ]  Yes

Type of change
- [ ]  New feature (non-breaking change which adds functionality)
- [ ]  Bug fix (non-breaking change which fixes an issue)
- [x]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ]  Super small fix (Corrected a typo, removed a comment, etc.)
- [x]  Added testing for: details page
- [ ]  Skip all the other stuff and briefly explain the fix.

Checklist:
- [x]  If this code needs to be tested, all tests are passing
- [x]  The code runs locally
- [x]  The code is DRY. If it's not, I tried my best
- [x]  I reviewed my code before pushing

What's next?

### CODE REVIEW

**Review by apete12 — APPROVED** [review]
awesome! thanks for making these updates.

**Review by aliceabarca — APPROVED** [review]
great job on the refactor for the pet details test.

**Review by judy0ye — APPROVED** [review]
looks good!

File: src/components/PetDetails/PetDetails.js [hunk 1]
```diff
@@ -18,13 +18,15 @@ function PetDetails({ allPets }) {
         <Link to={'/'}><button className='back-to-all-pets-button'>Back to All Pets</button></Link>
       </div>
       <section className='details-container'>
-        <div className='owner-info'>
+        <div className='pet'>
+          {/* this used to be .owner-info */}
```
Comment (judy0ye) [inline src/components/PetDetails/PetDetails.js:22]: thanks for the clarification!

**Pull request merged**